db/schema.rb

```diff
@@ -444,9 +444,4 @@
   add_index "versions", ["item_type", "item_id", "whodunnit"], name: "index_versions_on_item_type_and_item_id_and_whodunnit", using: :btree
   add_index "versions", ["team_id"], name: "index_versions_on_team_id", using: :btree
 
-  add_foreign_key "accounts", "teams"
```

Comment:
@caiosba any ida why these foreign keys keep popping in and out of existence seemingly randomly?

Reply:
no... PG version maybe? I get the same behavior as @amoedoamorim btw